app/src/components/ArtifactTimeline.tsx

```diff
@@ -0,0 +1,112 @@
+import { Center, ScrollArea, Space, Text, Timeline } from '@mantine/core';
+import { Message } from 'tabler-icons-react';
+import { useEffect, useState } from 'react';
+import ArtifactComments from './ArtifactComments';
+
+interface ArtifactCommentProps {
+  date: string | undefined;
+  body: string | undefined;
+  author: string | undefined;
+}
```

Comment:
I would just import the interface from `ArtifactComments.tsx` since they are the same.